packages/babel-plugin-transform-spread/test/fixtures/call-context/flow-type-cast/options.json

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
```

Comment:
Can you use syntax plugin only? So we can test the types are indeed preserved.
```
{
  "plugins": ["syntax-flow"]
}
```

Reply:
Good call! I tweaked that, and also did the same for the two other tests.